modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/request.json

```diff
@@ -1,48 +1,54 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "description": "Claims API 526 Schema",
-  "type": "object",
-  "additionalProperties": false,
-  "required": ["data"],
-  "properties": {
-    "data": {
-      "type":"object",
-      "required":["attributes"],
-      "properties": {
-        "attributes": {
-          "type":"object",
-          "required": [
-            "veteranIdentification",
-            "serviceInformation",
-            "disabilities",
-            "claimantCertification",
-            "claimProcessType"
-          ],
-          "properties": {
-            "claimDate": {
-              "description": "Date of claim submission to be applied to signature on form 21-526EZ. Date cannot be in the future.",
-              "pattern": "^(?:[0-9]{4})-(?:0[1-9]|1[0-2])-(?:0[1-9]|[1-2][0-9]|3[0-1])(?:T[0-1][0-9]:[0-5][0-9]:[0-5][0-9](?:[-+][0-9]{2}:[0-9]{2}|Z))*$",
-              "type": "string",
-              "example": "2018-06-04"
-            },
-            "claimProcessType": {
-              "type": "string",
-              "description": "Select type of claim program/process that applies to the Veteran.",
-              "enum": [
-                "STANDARD_CLAIM_PROCESS",
-                "FDC_PROGRAM",
-                "BDD_PROGRAM"
-              ]
-            },
-            "claimantCertification": {
-              "type": "boolean",
-              "description": "Claimant certifies and authorizes release of information.",
-              "default": false
-            },
-            "veteranIdentification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Should these be indented 4 or 2 spaces, the rest of the indentations (I.E. ln 6 -> ln 7) is 2 spaces.